api/DOMMatrix.json

```diff
@@ -40,16 +146,18 @@
           "spec_url": "https://drafts.fxtf.org/geometry/#dom-dommatrix-dommatrix",
           "support": {
             "chrome": {
-              "version_added": "61"
+              "version_added": "2"
```

Comment:
I don't think we should change the data for the constructor entry. Adding aliases that go back would be OK, and look just like the parent feature, but this data I would interpret as `new DOMMatrix()` being available since Chrome 2, which it isn't. The data isn't always shown with the parent feature after all:
https://developer.mozilla.org/docs/Web/API/DOMMatrix/DOMMatrix